### PR TITLE
ClassicSwap DEX and Crowdsale Announcement

### PR DESCRIPTION
**Pull request by ClassicSwapInc**
ClassicSwap DEX and Crowdsale Announcement

### CODE REVIEW

**Comment by IstoraMandiri** [comment]
Looks good, will approve once #693 is merged

**Review by gitr0n1n — APPROVED** [review]
LGTM

**Comment by w1g0** [comment]
Project canceled.

**Pull request closed without merging**
